src/config/mod.rs

```diff
@@ -13,35 +13,38 @@
 // see <https://www.gnu.org/licenses/>.
 use std::fs::{create_dir_all, File};
 use std::io::{Read, Write};
+use std::mem;
 use std::path::Path;
+use std::process::exit;
 
 use anyhow::{Context, Result};
 use serde::{Deserialize, Serialize};
 
+use crate::otlp::config::OtlpConfig;
+
 #[derive(Clone)]
 #[derive(Debug)]
 #[derive(Default)]
-#[derive(PartialEq, Eq, PartialOrd, Ord)]
+#[derive(PartialEq, Eq)]
```

Comment:
Shall we merge this traits into one line? 
```
#[derive(Clone, Debug, Default, PartialEq ...)]
```

Reply:
I think it would be more manageable to categorize the different derives.